loader/generated/vk_loader_extensions.c

```diff
@@ -1379,7 +1382,7 @@ void init_extension_device_proc_terminator_dispatch(struct loader_device *dev) {
        dispatch->CmdInsertDebugUtilsLabelEXT = (PFN_vkCmdInsertDebugUtilsLabelEXT)gpda(dev->icd_device, "vkCmdInsertDebugUtilsLabelEXT");
 #ifdef VK_USE_PLATFORM_WIN32_KHR
     // ---- VK_EXT_full_screen_exclusive extension commands
-    if (dev->extensions.ext_full_screen_exclusive_enabled && dev->extensions.khr_device_group_enabled)
+    if (dev->extensions.ext_full_screen_exclusive_enabled)
```

Comment:
Why did this change?

Reply:
https://gitlab.khronos.org/vulkan/vulkan/-/merge_requests/5542

Looks like the loader codegen needs to account for the extension dependency schema updates